### PR TITLE
Quickstart touchups

### PR DESCRIPTION
**Pull request by AliceProxy**
Signed-off-by: AliceProxy <alicewasko@datawire.io>

Couple small edits to the formatting of the quickstart doc.

Prioritize landing https://github.com/envoyproxy/gateway/pull/595 and then I'll rebase when that gets merged.

### CODE REVIEW

**Comment by danehans** [comment]
@AliceProxy #595 has landed, so feel free to update this PR.

File: docs/user/QUICKSTART.md [hunk 1]
```diff
@@ -16,39 +16,43 @@ Install the Gateway API CRDs:
...
 kubectl apply -f https://raw.githubusercontent.com/envoyproxy/gateway/v0.2.0-rc2/examples/kubernetes/httpbin.yaml
 ```
 
-The Gateway API resources must be created in the following order. First, create the GatewayClass:
+The Gateway API resources must be created in the following order:
 
-```shell
-kubectl apply -f https://raw.githubusercontent.com/envoyproxy/gateway/v0.2.0-rc2/examples/kubernetes/gatewayclass.yaml
-```
+1. First, create a GatewayClass:
```
Comment (chauhanshubham) [inline docs/user/QUICKSTART.md:33]: The quickstart related yaml was squashed into a single yaml, so we'll just need to apply that to create gwclass, gw, httproute, service, deployment.

**Review by LukeShu — CHANGES_REQUESTED** [review]
What Shubham said, those YAML locations are no longer correct.

**Review by Xunzhuo — COMMENTED** [review]
Plz rebase with main, to get this updated with the latest changes, and after that, let us see what needs to be updated:)

**Comment by AliceProxy** [comment]
With the new quickstart updates I don't feel like this is needed anymore

**Pull request closed without merging**